.github/workflows/release.yml

```diff
@@ -25,8 +25,8 @@ jobs:
         id: get_version
         run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
 
-      - name: Build without tests
-        run: ./gradlew build -PreleaseVersion=${{ steps.get_version.outputs.VERSION }} -x test -x spotlessCheck
+      - name: Build artifacts
+        run: ./gradlew artifacts -PreleaseVersion=${{ steps.get_version.outputs.VERSION }}
```

Comment:
Ok, I understand.  On a release workflow, you want to run the tests to ensure the release is safe.  The only issue I see here is that this takes quite a while to run through.  In fact, I don't know if its even possible on a local machine ?

Reply:
Hmmm, so I guess that doesn't actually matter!!  Its going to be running the CI pipeline.  But, still I'm not clear exactly which of the various test suits it will run now.  I'll check this for my own sanity.

Reply:
No tests are run by this, it only build the jar and the zip